### PR TITLE
Implement tpx command to accept sequence of bytes

### LINKED ISSUE #4361: Show value for type for some specific bytes
(from xvilka)
Add one more pls - 'ability to apply type to the sequence of bytes directly from command line'

`ts struct qwe 474333456`

(pancake)
it is useful to be able to show the values for a specific type (struct, union ..) with some specific data.
this can be already done with `pf ... @x:....` .. the pf expression can be extracted from the struct type, and the hexpairs will go after the @x: .. maybe we need a specific command to avoid having to use @x: and so on

### PR DESCRIPTION
**Pull request by sivaramaaa**
Fixes #4361 , #6091

* Honors @x: in pf

* Implemented tpx command to accept sequence of bytes and fill that into loaded type

Added test in r2r [#1320](https://github.com/radare/radare2-regressions/pull/1320)

### CODE REVIEW

File: libr/core/cmd_print.c [hunk 1]
```diff
@@ -1036,12 +1036,15 @@ static void cmd_print_format(RCore *core, const char *_input, const ut8* block,
 		/* This make sure the structure will be printed entirely */
 		char *fmt = input + 1;
 		while (*fmt && IS_WHITECHAR (*fmt)) {
-			fmt++;
-		}
-		int size = r_print_format_struct_size (fmt, core->print, mode, 0) + 10;
-		if (size > core->blocksize) {
-			r_core_block_size (core, size);
-		}
+      fmt++;
```
Comment (radare) [inline libr/core/cmd_print.c]: fuck github editor for not supporting tabs

File: libr/core/cmd_print.c [hunk 2]
```diff
@@ -1036,12 +1036,15 @@ static void cmd_print_format(RCore *core, const char *_input, const ut8* block,
...
+    }
+		int struct_sz = r_print_format_struct_size (fmt, core->print, mode, 0) + 10;
+		int size = R_MAX (core->blocksize, struct_sz);
+		ut8 *buf = calloc (1, size);
```
Comment (radare) [inline libr/core/cmd_print.c]: can be null, check for it

File: libr/core/cmd_type.c [hunk 3]
```diff
@@ -786,28 +787,30 @@ static int cmd_type(void *data, const char *input) {
...
-			char *ptr = strchr (type, '=');
-			if (ptr) {
-				char *tmp = ptr-1;
-				*ptr++ = 0;
-				while(isspace (*ptr)) ptr++;
-				while(isspace (*tmp)) *tmp-- = 0;
-				addr = r_num_math (core->num, ptr);
-			}
+	case 'p': { // "tp"
+		char *ptr = strdup (input);
```
Comment (radare) [inline libr/core/cmd_type.c]: can be null

File: libr/core/cmd_print.c (hunk 1 above)
Comment (sivaramaaa) [reply libr/core/cmd_print.c]: np , i will fix it :)

**Pull request merged**